projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts

```diff
@@ -103,6 +92,7 @@ export class TuiInputCardGroupedComponent
     @ViewChild('inputCVC', {read: TuiAutoFocusDirective})
     private readonly cvcCardAutofocusRef?: TuiAutoFocusDirective;
 
+    private readonly el = inject(ElementRef<HTMLElement>);
```

Comment:
Computed type is still `ElementRef<any>`
<img width="829" alt="element-ref" src="https://github.com/taiga-family/taiga-ui/assets/35179038/4178e85a-3cc2-4316-b36d-4bf5e354789c">

Did you mean the following?
```ts
private readonly el: ElementRef<HTMLElement> = inject(ElementRef);
```

Reply:
fixed